sdks/js/3.x/docs/APIReference.md

```diff
@@ -6,6 +6,11 @@
 
 #### configure
 
+> **_Note:_** 
+This method is for Media SDK JavaScript v3.x.  
+For Media Analytics v3.x for Audio and Vidoe extension, configure the setup via data collection (Tags).
```

Comment:
`For Media SDK Javascript 3.X with Tags extension, setup the configuration via Data Collection UI (Tags UI)`